pkg/proxy/ipvs/proxier.go

```diff
@@ -286,7 +299,7 @@ func NewProxier(ipt utiliptables.Interface,
 	exec utilexec.Interface,
 	syncPeriod time.Duration,
 	minSyncPeriod time.Duration,
-	excludeCIDRs []string,
+	excludeCIDRStrs []string,
```

Comment:
I don't think this variable name needed to be updated. `excludeCIDRs` IMO is fine